space_packet_parser/xarr.py

```diff
@@ -123,7 +123,7 @@ def _get_minimum_numpy_datatype(
 
 def create_dataset(
     packet_files: Union[ReadableBinaryPacket, Iterable[ReadableBinaryPacket]],
-    xtce_packet_definition: Union[str, Path, definitions.XtcePacketDefinition],
+    xtce_packet_definition: Union[str, Path, PathLike, definitions.XtcePacketDefinition],
```

Comment:
If we are using PathLike, I think we can get rid of `str` and `Path` references then because they are contained within this definition aren't they?
```suggestion
    xtce_packet_definition: Union[PathLike, definitions.XtcePacketDefinition],
```